components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/dto/TierDTO.java

```diff
@@ -10,64 +12,132 @@
 
 @ApiModel(description = "")
 public class TierDTO  {
-  
-  
-  @NotNull
-  private String name = null;
-  
-  
-  private String rate = null;
-  
-  
-  private String roles = null;
-
-  
-  /**
-   **/
-  @ApiModelProperty(required = true, value = "")
-  @JsonProperty("name")
-  public String getName() {
-    return name;
-  }
-  public void setName(String name) {
-    this.name = name;
-  }
-
-  
-  /**
-   **/
-  @ApiModelProperty(value = "")
-  @JsonProperty("rate")
-  public String getRate() {
-    return rate;
-  }
-  public void setRate(String rate) {
-    this.rate = rate;
-  }
-
-  
-  /**
-   **/
-  @ApiModelProperty(value = "")
-  @JsonProperty("roles")
-  public String getRoles() {
-    return roles;
-  }
-  public void setRoles(String roles) {
-    this.roles = roles;
-  }
-
-  
-
-  @Override
-  public String toString()  {
-    StringBuilder sb = new StringBuilder();
-    sb.append("class TierDTO {\n");
-    
-    sb.append("  name: ").append(name).append("\n");
-    sb.append("  rate: ").append(rate).append("\n");
-    sb.append("  roles: ").append(roles).append("\n");
-    sb.append("}\n");
-    return sb.toString();
-  }
+
+
+    @NotNull
+    private String name = null;
+
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.